README.md

```diff
@@ -56,6 +56,22 @@ and health-checking mechanisms.
   ```sh
   sudo yum localinstall -y https://yum.puppetlabs.com/puppet/el/7/x86_64/puppet-agent-7.0.0-1.el7.x86_64.rpm
   ```
+   * **Note:**   
+     Check facter version details, if already installed / exist in system
```

Comment:
We can move the checking and removal part before the original puppet installation command :
`sudo yum localinstall -y https://yum.puppetlabs.com/puppet/el/7/x86_64/puppet-agent-7.0.0-1.el7.x86_64.rpm`
and we can remove the installation of facter command (`ln -s /opt/puppetlabs/bin/facter /usr/bin/facter`), as it should get installed with puppet installation.

Reply:
@suvratjoshi  agreed with first point of moving these steps (except last linking step) before puppet installation command. 
For second point, this does not happen when we install puppet agent so this step is required.  @pavankrishnat please move the steps before puppet installtion except last step.